### DISCUSSION TITLE
Lesson 7: Getting error while trying to verify the contract

### DISCUSSION THREAD

**Discussion by cell0901** [Cyfrin/foundry-full-course-cu#1463] [General]
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/142923689/f8c55484-14d6-4392-8593-cec1d910e220)

getting this error while trying to verify the contract through the Etherscan api key, i have doubled checked the api key its correct and  make file command is also seems good

**Comment by EngrPips** [comment]
Hello @cell0901 , I am not sure why you are getting that error and i run into that same error myself few days ago, I will check and reach back if i found anything that could help us resolve that error.
